Automattic/jetpack
# Added an additional post content sync call before publishing.

Fixes a problem that Jetpack might encounter when Sync actions are skipped for whatever reason before a post gets published.

#### Changes proposed in this Pull Request:

* Added a new `wp_insert_post` call to Posts module.

#### Other information:

#### Jetpack product discussion


* 7411-gh-Automattic/jpop-issues

#### Does this pull request change what data or activity we track or use?


Nope.

#### Testing instructions:





* Testing this one is tricky, but you can try emulating skipped Sync events by making the `wp_insert_post` method of the Posts module short circuit on every call but the one that happens at publish time. I did that by introducing a fourth argument into that method just for testing that would always be false instead of that time where we add it. Hopefully that makes sense.
* Try to publish a draft using that modified setup, see that the post doesn't get published.
* Try to publish a draft with the added call, see that it's published correctly.
